buildSrc/src/main/groovy/BuildkitBuild.groovy

```diff
@@ -0,0 +1,352 @@
+            }
+        }
+
+        jdbcDriverFiles.forEach {
+            if (Files.exists(Paths.get(it.path))) {
+                Files.copy(Paths.get(it.path), Paths.get("${driverDir.toString()}/$it.name"), StandardCopyOption.REPLACE_EXISTING)
+            }
+        }
+
+        if (!containerRepo.get().isEmpty()) {
+            targetRepo = "${containerRepo.get()}"
+            if (!containerTag.get().isEmpty()) {
+                targetTags = ["${containerTag.get()}"]
+                imageRepo.add([name: targetRepo, tag: targetTags])
+            } else {
+                imageRepo.add([name: targetRepo, tag: ""])
+            }
+        } else if (dockerHubPublish.get()) {
+            targetRepo = "corda"
+            targetTags = ["${version}"]
+            imageRepo.add([name: targetRepo, tag: targetTags])
+        } else if (preTest.get()) {
+            targetRepo = "corda-os-docker-pre-test.software.r3.com"
+            targetTags = ["preTest-${tagPrefix}${version}", "preTest-${tagPrefix}${gitRevision}"]
+            imageRepo.add([name: targetRepo, tag: targetTags])
+        } else if (releaseType == 'RC' || releaseType == 'GA') {
+            targetRepo = "corda-os-docker-stable.software.r3.com"
+            targetTags = ["${tagPrefix}latest", "${tagPrefix}${version}"]
+            imageRepo.add([name: targetRepo, tag: targetTags])
+        } else if (releaseType == 'BETA' && !nightlyBuild.get()) {
+            targetRepo = "corda-os-docker-unstable.software.r3.com"
+            targetTags = ["${tagPrefix}unstable", "${tagPrefix}${gitRevision}", "${version}"]
+            imageRepo.add([name: targetRepo, tag: targetTags])
+        } else if (releaseType == 'ALPHA' && !nightlyBuild.get()) {
+            targetRepo = "corda-os-docker-dev.software.r3.com"
+            targetTags = ["${tagPrefix}${gitRevision}", "${version}"]
+            imageRepo.add([name: targetRepo, tag: targetTags])
+        } else if (releaseType == 'BETA' && nightlyBuild.get()) {
+            targetRepo = "corda-os-docker-nightly.software.r3.com"
+            targetTags = ["${tagPrefix}nightly", "${tagPrefix}nightly-${timeStamp}"]
+            imageRepo.add([name: targetRepo, tag: targetTags])
+        } else if (releaseType == 'ALPHA' && nightlyBuild.get()) {
+            targetRepo = "corda-os-docker-nightly.software.r3.com"
+            targetTags = ["${tagPrefix}nightly-${version}", "${tagPrefix}nightly-${gitRevision}"]
+            imageRepo.add([name: targetRepo, tag: targetTags])
+        } else {
+            targetRepo = "corda-os-docker-dev.software.r3.com"
+            targetTags = ["latest-local", "${version}", "${gitRevision}"]
+            imageRepo.add([name: targetRepo, tag: targetTags])
+        }
+
+        List<String> javaArgs = new ArrayList<String>(arguments.get())
+        javaArgs.add("-Dlog4j2.debug=\${ENABLE_LOG4J2_DEBUG:-false}")
+        javaArgs.add("-Dlog4j.configurationFile=log4j2-console.xml")
```

Comment:
Same here, we do not have `log4j2-console.xml` in this project.
`app/src/main/resources/log4j2.xml` controls logging configuration.

Reply:
Yeah I will remove this now

Reply:
unnecessary args removed